src/index.html

```diff
@@ -1,18 +1,44 @@
-<!doctype html>
+<!DOCTYPE html>
 <html lang="en">
-  <head>
-    <meta charset="UTF-8" />
-    <meta
-      name="viewport"
-      content="width=device-width, initial-scale=1.0"
-    />
-    <title>Calendar</title>
-    <link
-      rel="stylesheet"
-      href="styles/index.scss"
-    />
-  </head>
+<head>
+  <meta charset="UTF-8" />
+  <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+  <title>Calendar</title>
+  <link rel="stylesheet" href="./styles/index.scss" />
```

Comment:
The stylesheet link references a `.scss` file. Typically, SASS files need to be compiled to CSS before they can be used in an HTML document. Make sure to compile the SASS file and link the resulting CSS file instead.